astronomer/providers/amazon/aws/example_dags/example_create_emr_on_eks_cluster.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# create cluster
+eksctl create cluster \
```

Comment:
Please format this by adding tab between line 5 to 11

Reply:
Added